debian/changelog

```diff
@@ -1,5 +1,8 @@
 cloud-init (22.1-43-gbf94945f-0ubuntu1~20.04.1) UNRELEASED; urgency=medium
```

Comment:
Not sure if I got the version right here - does this one need 20.04.1 bumped to 20.04.4? Or is fine as is due to the snapshot previously bumping 22.1-14 -> 22.1-43

Reply:
All good here thanks for the question. Since we had already performed `new-upstream-snapshot` into `ubuntu/focal` and not yet officially released this branch to 22.04 we can and should append to this "open" UNRELEASED changelog entry without needing to bump the version. +1

Reply:
For our "process" the d/changelog entry becomes closed when we label UNRELEASED -> <some_series_name> to indicate to us that we have published this version to official Ubuntu -updates repos.